code/modules/mob/living/carbon/carbon.dm

```diff
@@ -315,6 +315,9 @@
 	if(.)
 		if(visual)
 			return
+		if(weakeyes)
```

Comment:
this will stun weakeye races even if their eyes are ripped out
which is hilarious but probably not intended